internal/commands/app/create.go

```diff
@@ -134,14 +146,27 @@ func (cmd *CommandCreate) Handler(profile *cli.Profile, ui terminal.UI, clients
 		default:
 			dataSourceDir = local.NameDataSources
 		}
-		data, err := local.MarshalJSON(ds)
-		if err != nil {
-			return err
+		if dsCluster.Name != "" {
+			data, err := local.MarshalJSON(dsCluster)
+			if err != nil {
+				return err
+			}
+			path := filepath.Join(dataSourceDir, dsCluster.Name, local.FileConfig.String())
+			err = local.WriteFile(filepath.Join(dir, path), 0666, bytes.NewReader(data))
```

Comment:
[no action needed here] if/when we ever need to formalize these data source definitions (read: move them to `local`), lets consider making a `DataSource` interface that implements a `Name() string` getter.  with that, we'd be able to have a single function here `func writeDataSourceConfig(dataSource local.DataSource)` (or in the `local` package itself)

again, this is good as is here, don't want to go type-crazy and this logic isn't changing much anytime soon.  just food for thought